flopy/modpath/mp6sim.py

```diff
@@ -502,7 +496,7 @@ def write_file(self, data=None, float_format="{:.8f}"):
         data["k0"] += 1
         data["i0"] += 1
         data["j0"] += 1
-        if pd is not None and self.use_pandas and len(data) > 0:
```

Comment:
While this adds a harder `pandas` dependency that seems OK to me. Could consider bringing it into the core as we use it in [more places than the other optional dependencies](https://github.com/modflowpy/flopy/blob/develop/docs/flopy_method_dependencies.md)? By GH stats it is also almost as widely used as `numpy`

If it is to remain optional, maybe worth updating the ctor default to `use_pandas=False`. And the docstring still reads "if True and pandas is available" but below we now throw an error if True and no pandas

Reply:
I'll agree to change default to `use_pandas=False`, so will add that here. See https://github.com/modflowpy/flopy/discussions/1788 for a discussion on moving pandas as a core dependency.